index.html

```diff
@@ -290,6 +290,39 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
+    <div class="main">
```

Comment:
[OPTIONAL] To improve the markup semantics and its accessibility, then I will strongly advise you use a footer tag here instead of the div tag.